recipe/meta.yaml

```diff
@@ -1,27 +1,29 @@
 package:
   name: travis-encrypt
-  version: "0.7.2"
+  version: "1.0.0"
 
 source:
-  url: https://files.pythonhosted.org/packages/63/ed/71a981d4aa842fb0ed827bcc0560c09392635210497d425b70452c7f3d14/travis-encrypt-0.7.2.tar.gz
-  sha256: 185a3a8a01be2fa4b88b7b2a0b7bbf8fbbe684f1e154cfbafcd4bd3d1bd01f94
+  url: https://files.pythonhosted.org/packages/ff/48/62e01fbc7a4090cb1298b3027fa81080c110c28ed99a8f98d58fc87e5de6/travis-encrypt-1.0.0.tar.gz
```

Comment:
Could we template this URL?

ref: https://github.com/conda-forge/staged-recipes/blob/641b3536c78f913c6fb24e957c520923b7848c81/recipes/example/meta.yaml#L14